docroot/themes/custom/dcp/sass/components/_footer.scss

```diff
@@ -0,0 +1,65 @@
+.region-footer {
+  min-height: 390px;
+  background-image: url("../images/footer_image.png");
+  background-size: 100% 100%;
+  padding: 60px;
+  display: flex;
+
+  [class*="menu--"] {
```

Comment:
@theyogendraDanwar - can we use menu attribute module?

Reply:
@vidit-anjaria this is only for the footer region it doesn't make sense to me add a whole module just for the footer for a small section if in future there are multiple requirements then we can add that

Reply:
@vidit-anjaria this is only for the footer region it doesn't make sense to me add a whole module just for the footer for a small section if in future there are multiple requirements then we can add that

Reply:
@theyogendraDanwar - I would suggest to use the menu attribute module though.